src/server.ts

```
@@ -35,6 +36,12 @@
         reply.code(400).send({error: 'Invalid request'});
         return;
       }
+      
+      let feesValid = await feesTooHigh(result.data);
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)